tests/HttpCache/VarnishPurgerTest.php

```diff
@@ -118,48 +118,48 @@ public function provideChunkHeaderCases()
         yield 'one iri' => [
             50,
             ['/foo'],
-            ['(^|\,)/foo($|\,)'],
+            ['/foo($|\,)'],
         ];
 
         yield 'few iris' => [
             50,
             ['/foo', '/bar'],
-            ['((^|\,)/foo($|\,))|((^|\,)/bar($|\,))'],
+            ['(/foo|/bar)($|\,)'],
         ];
 
         yield 'iris to generate a header with exactly the maximum length' => [
             56,
             ['/foo', '/bar', '/baz'],
-            ['((^|\,)/foo($|\,))|((^|\,)/bar($|\,))|((^|\,)/baz($|\,))'],
+            ['(/foo|/bar|/baz)($|\,)'],
         ];
 
         yield 'iris to generate a header with exactly the maximum length and a smaller one' => [
             37,
             ['/foo', '/bar', '/baz'],
             [
-                '((^|\,)/foo($|\,))|((^|\,)/bar($|\,))',
-                '(^|\,)/baz($|\,)',
+                '(/foo|/bar)($|\,)',
```

Comment:
I think something needs to be changed in the splitting logic: the 2 segments would now fit the 37 characters limit if merged.

Reply:
hi @julienfalque, I'm fixing tests right now.